src/reporters/json.ts

```diff
@@ -71,6 +84,30 @@ type OutputFields = {
   blob_mime?: string;
 };
 
+type ScreenshotBlob = {
+  blob: string;
+  id: string;
+};
+
+type ScreenshotReference = {
+  width: number;
+  height: number;
+  blockWidth: number;
```

Comment:
Can we remove the `blockWidth` and `blockHeight` attributes here and put them on a per block level in the array below. That way in the future we can have irregularly size blocks and do sophisticated compositing on the client side. We can have the agent to generate whatever arrangement of blocks makes sense going far beyond a simple grid.